src/core/Microsoft.PowerToys.Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -311,26 +312,37 @@
                     />-->
         </StackPanel>
 
-        <StackPanel 
-            x:Name="SidePanel"
-            Orientation="Vertical"
-            HorizontalAlignment="Left"
-            Width="{StaticResource SidePanelWidth}"
-            Grid.Column="1">
-
-            <TextBlock
-                x:Uid="About_KeyboardManager"
-                Style="{StaticResource SettingsGroupTitleStyle}"
-                Margin="{StaticResource XSmallBottomMargin}"
-                Foreground="{Binding Mode=TwoWay, Path=TextColor}"/>
+        <RelativePanel x:Name="SidePanel" 
+                    HorizontalAlignment="Left"
+                    Width="{StaticResource SidePanelWidth}"
+                    Grid.Column="1">
+            <StackPanel x:Name="DescriptionPanel">
+                <TextBlock x:Uid="About_KeyboardManager" x:Name="AboutTitle" Grid.ColumnSpan="2"
```

Comment:
we can use some formatting here, put each attribute in a separate line!

Reply:
Will fix